core/server/utils/index.js

```diff
@@ -19,13 +19,14 @@ utils = {
     /**
      * Timespans in seconds and milliseconds for better readability
      */
-    ONE_HOUR_S: 3600,
-    ONE_DAY_S: 86400,
-    ONE_YEAR_S: 31536000,
-    ONE_HOUR_MS: 3600000,
-    ONE_DAY_MS: 86400000,
-    ONE_MONTH_MS: 2628000000,
-    ONE_YEAR_MS: 31536000000,
+    ONE_HOUR_S:          3600,
+    ONE_DAY_S:          86400,
+    ONE_YEAR_S:      31536000,
+    ONE_HOUR_MS:      3600000,
+    ONE_DAY_MS:      86400000,
+    ONE_WEEK_MS:    604800000,
+    ONE_MONTH_MS:  2628000000,
+    ONE_YEAR_MS:  31536000000,
```

Comment:
I was gonna add a comment to say I'll undo this if people hate it. My OCD got the better of me sorry :flushed: